packages/vite/src/executors/test/schema.json

```
@@ -56,6 +56,10 @@
     "reportsDirectory": {
       "type": "string",
       "description": "Directory to write coverage report to."
+    },
+    "testFile": {
+      "description": "The name of the file to test.",
+      "type": "string"
```

the option is called `testNamePattern` (aliased to `-t`) in the vitest cli docs
we should use that name instead as the executor options should closely mimic the CLI of the tool they are wrapping.

https://vitest.dev/guide/cli.html#options

```suggestion
      "testFilePattern": {
        "aliases": ["t"],
        "description": "Run tests with full names matching the pattern",
        "type": "string"
```